BoxContentSDK/BoxContentSDK/Requests/BOXFolderItemsRequest.m

```diff
@@ -95,6 +100,8 @@ - (void)performRequestWithCached:(BOXItemsBlock)cacheBlock refreshed:(BOXItemsBl
         NSMutableArray *results = [[NSMutableArray alloc] init];
 
         BOXItemsBlock localRefreshBlock = ^(NSArray *items, NSError *error){
+            self.paginatedRequest = nil;
```

Comment:
should we call this as first thing in recursiveFetch/paginatedRefreshItemFetch block instead of in localRefreshBlock?